src/com/palmergames/bukkit/towny/event/town/TownTrustTownAddEvent.java

```diff
@@ -0,0 +1,39 @@
+package com.palmergames.bukkit.towny.event.town;
+
+import com.palmergames.bukkit.towny.event.CancellableTownyEvent;
+import com.palmergames.bukkit.towny.object.Town;
+import com.palmergames.bukkit.towny.object.Translation;
+import org.bukkit.command.CommandSender;
+import org.jetbrains.annotations.NotNull;
+
+public class TownTrustTownAddEvent extends CancellableTownyEvent {
```

Comment:
```suggestion
public final class TownTrustTownAddEvent extends CancellableTownyEvent {
```
class is already finally defined so class can be finalized